sonar-groovy-plugin/src/main/java/org/sonar/plugins/groovy/jacoco/AbstractAnalyzer.java

```diff
@@ -140,32 +145,6 @@ public final void readExecutionData(File jacocoExecutionData, SensorContext cont
     }
   }
 
-  private CoverageBuilder analyze(ExecutionDataStore executionDataStore) {
-    CoverageBuilder coverageBuilder = new CoverageBuilder();
-    Analyzer analyzer = new Analyzer(executionDataStore, coverageBuilder);
-    for (File binaryDir : moduleFileSystem.binaryDirs()) {
-      analyzeAll(analyzer, binaryDir);
-    }
-    return coverageBuilder;
-  }
-
-  /**
-   * Copied from {@link Analyzer#analyzeAll(File)} in order to add logging.
-   */
-  private static void analyzeAll(Analyzer analyzer, File file) {
-    if (file.isDirectory()) {
-      for (File f : file.listFiles()) {
-        analyzeAll(analyzer, f);
-      }
-    } else if (file.getName().endsWith(".class")) {
-      try {
-        analyzer.analyzeAll(file);
-      } catch (Exception e) {
-        JaCoCoExtensions.logger().warn("Exception during analysis of file " + file.getAbsolutePath(), e);
-      }
-    }
-  }
-
   private static CoverageMeasuresBuilder analyzeFile(ISourceFileCoverage coverage) {
     CoverageMeasuresBuilder builder = CoverageMeasuresBuilder.create();
     for (int lineId = coverage.getFirstLine(); lineId <= coverage.getLastLine(); lineId++) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the total number of break and continue statements in this loop to use at most one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS135)